docs/classic/checkers.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: "docu"
+title: "Checkers"
+actions: "Discrete"
+agents: "2"
+manual-control: "No"
+action-shape: "Discrete(8*8*4)"
+action-values: "Discrete(8*8*4)"
+observation-shape: "Box(8, 8, 4)"
+observation-values: "[0, 1]"
+num-states: "10^21"
```

Comment:
I thought we weren't documenting num_states.

Reply:
No clue, it was in some of the other classic documentation so I looked it up. I feel like if we have the information, it's a useful thing to know.